server/translations.ts

```diff
@@ -141,22 +142,45 @@ function getDocument(filePath) {
     };
   }
 
+  function getCommitBehindFromLatest(filename, commitHash) {
```

Comment:
I get the index of the file's latest hash list like this and see how far behind it is. However, it is slow. For the PoC, I wrote it as it is at first, but I need another method.